driver-portal/src/ClientApp/src/app/letters-to-driver/letters-to-driver.component.ts

```diff
@@ -59,9 +60,13 @@ export class LettersToDriverComponent implements OnInit {
     this.filteredDocuments = this._letterDocuments?.slice(0, this.pageSize);
   }
 
+  letterOutDocuments: Document[] = [];
+
   ngOnInit(): void {
     window.scrollTo(0, 0);
+    this.getLetterOutDocument();  
```

Comment:
last change, just need to do the same thing you did here to partner portal "letters-to-driver.component.ts", delete the userService, driverId parameter and driverDetails variable

Reply:
I adjusted the changes on letters to driver similar to driver portal